docs/language.rst

```diff
@@ -0,0 +1,170 @@
++------------------------+-------------------+-------------------+---------------------+
+|                                           Other                                      |
++========================+===================+===================+=====================+
+| ~                      | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| ->                     | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| FOR_NETCONS            | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| LOCAL                  | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| TITLE                  | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| DEFINE                 | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| INCLUDE                | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| SWEEP                  | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| PLOT                   | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| CONDUCTANCE            | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| PROTECT                | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| FROM                   | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| WATCH                  | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| MUTEXLOCK              | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
+| MUTEXUNLOCK            | yes               | yes               | no                  |
++------------------------+-------------------+-------------------+---------------------+
```

Comment:
I don't think we generate anything for mutex lock/unlock? But I think it's not needed because we don't use pthread (or other) based parallelisation anyway.